packages/stores/types.ts

```diff
@@ -234,11 +247,36 @@ type TUiStore = {
     toggleLanguageSettingsModal: () => void;
     toggleReadyToDepositModal: () => void;
     toggleSetCurrencyModal: () => void;
+    removeToast: () => void;
     is_ready_to_deposit_modal_visible: boolean;
+    reports_route_tab_index: number;
+    should_show_cancellation_warning: boolean;
+    toggleCancellationWarning: () => void;
+    toggleUnsupportedContractModal: () => void;
+    toggleReports: () => void;
```

Comment:
```suggestion
    toggleReports: (value: boolean) => void;
```
@mahdiyeh-deriv Same 👀